package.json

```diff
@@ -76,14 +76,14 @@
     "lodash": "^4.17.4",
     "lodash-compat": "^3.10.0",
     "nomnoml": "^0.3.0",
-    "rollup": "^1.19.4",
+    "rollup": "^2.34.2",
     "shelljs": "^0.8.2",
     "sinon": "^8.1.1",
     "url-toolkit": "^2.1.3",
     "videojs-contrib-eme": "^3.8.0",
     "videojs-contrib-quality-levels": "^2.0.4",
     "videojs-generate-karma-config": "~7.0.0",
-    "videojs-generate-rollup-config": "~5.0.1",
+    "videojs-generate-rollup-config": "~6.0.2",
```

Comment:
had to update to the newest videojs-generate-rollup-config so that the `multi-entry` plugin supports include/exclude.